src/flownet/templates/webviz_ahm_config.yml.jinja2

```
@@ -32,7 +32,7 @@ pages:
         {%- for iteration in [iterations[0], iterations[-1]] %}
          - iter-{{ iteration }}
         {%- endfor %}
-        obsfile: ./observations.yamlobs
+        obsfile: ./observations_training.yamlobs
```

The time series plots should be shown with the complete observations file (this is where the labels will be shown)

```suggestion
        obsfile: ./observations_complete.yamlobs
```